Tools/PostProcessing/RunsScan/EndPropPlots.py

```diff
@@ -0,0 +1,232 @@
+                  + '/' + path_hdf5 + '-' + sim + '.hdf5')
+    print('Now downloading data from: ', path_file)
+
+    fin=h5py.File(path_file,'r')
+    datasets=list(fin.keys())
+    print(datasets)
+
+    d_data={}
+    for ds in datasets:
+        d_data[ds]={}
+        d_data[ds]['units'] = fin[ds].attrs['units']
+        d_data[ds]['label'] = fin[ds].attrs['label']
+        d_data[ds]['range'] = fin[ds].attrs['range']
+        d_data[ds]['data']=fin[ds].value
+
+    fin.close()
+
+    return d_data
+
+d_z      = {}
+d_totq   = {}
+d_totp   = {}
+d_emean  = {}
+d_estd   = {}
+d_emitpx = {}
+d_emitx  = {}
+d_emitpy = {}
+d_emity  = {}
+d_bwdx   = {}
+d_bwdy   = {}
+d_bwdux   = {}
+d_bwduy   = {}
+
+for sim in sim_list:
+
+    d_data = read_sim_hdf5(sim)
+
+    d_z     [sim] = d_data['z']['data']
+    d_totq  [sim] = d_data['totq']['data']
+    d_totp  [sim] = d_data['totp']['data']
+    d_emean [sim] = d_data['emean']['data']
+    d_estd  [sim] = d_data['estd']['data']
+    d_emitpx[sim] = d_data['emitpx']['data']
+    d_emitx [sim] = d_data['emitx']['data']
+    d_bwdx  [sim] = d_data['xsig']['data']
+    d_bwdux [sim] = d_data['uxsig']['data']
+    if (if3d == 1):
+        d_emitpy[sim] = d_data['emitpy']['data']
+        d_emity [sim] = d_data['emity']['data']
+        d_bwdy  [sim] = d_data['ysig']['data']
+        d_bwduy [sim] = d_data['uysig']['data']
+
+
+
+# Plots settings
+STAND_SIZE=30
+plt.rc('font', size=STAND_SIZE)          # controls default text sizes
+plt.rc('axes', titlesize=STAND_SIZE)     # fontsize of the axes title
+plt.rc('axes', labelsize=STAND_SIZE)     # fontsize of the x and y labels
+plt.rc('xtick', labelsize=STAND_SIZE)    # fontsize of the tick labels
+plt.rc('ytick', labelsize=STAND_SIZE)    # fontsize of the tick labels
+plt.rc('legend', fontsize=STAND_SIZE)    # legend fontsize
+plt.rc('figure', titlesize=STAND_SIZE)   # fontsize of the figure title
+
+def p_par_evol(fname):
+    plt.figure(figsize=(24,6)) #12))
+
+    ax0=plt.subplot(1,3,1)
+    ax0.set_ylabel('Mean energy (GeV)')
+    ax1=plt.subplot(1,3,2)
+    ax1.set_ylabel('Energy spread (%)')
+    ax2=plt.subplot(1,3,3)
+    ax2.set_ylabel('Points per beam width')
+#     ax3=plt.subplot(2,3,4)
+#     ax3.set_ylabel('Emittance in x (um)')
+#     ax4=plt.subplot(2,3,5)
+#     ax4.set_ylabel('Emittance in y (um)')
+#     ax5=plt.subplot(2,3,6)
+#     ax5.set_ylabel('Charge [pC]') #(R < 3RMS)
+
+    arr_fdtd = np.asarray([i for i in range(nsims) if 'FDTD' in sim_list[i]])
+    c_fdtd = '#1f77b4'
+    s_fdtd = 'o'
+    size_fdtd = 300
+    arr_psatd = np.asarray([i for i in range(nsims) if 'FDTD' not in sim_list[i]])
+    c_psatd = '#ff7f0e'
+    s_psatd = 'P'
+    size_psatd = 250
+
+    x=np.asarray([ppl[sim] for sim in sim_list])
+    y0=np.asarray([d_emean[sim][-1] for sim in sim_list])
+    y1=np.asarray([d_estd[sim][-1] for sim in sim_list])
+    y2=np.asarray([ppw[sim] for sim in sim_list])
+#     y3=np.asarray([d_emitx[sim][-1] for sim in sim_list])
```

Comment:
Do you potentially want to remove the commented code or `if ...:` it with an option? :)

Reply:
Hi @ax3l !
No, no.
These scripts are mere examples of what users can do for analysis - that they should adapt to their problems and desired analysis/plots.
And, of course, for us to use for our milestones!
I want to keep the line for users to be able to uncomment and use if they want.
But only after they went through the script.